participants/kumusic.md

```diff
@@ -0,0 +1,69 @@
+# <KuMusic>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+**Our project name is KuMusic, I'm simon in Slack.**
+
+## (1) Brief Project Description
+
+- A free music platform, providing non copyright music appreciation
+- Everyone can analyze their own voice
+
+## (2) Link to Project UI
+
+- [ui](https://github.com/Thchoonlophon/kumusic/tree/master/ui)
+
+
+## (3) What does your application/UI do?
+
+Users can upload their voice and listen online.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+**Music, voice**
```

Comment:
Can you specify which curated datasets you're using?  Is it one from the /datasets.md list?